pkg/framework/aries/build_interop.go

```diff
@@ -0,0 +1,82 @@
+// +build ACAPyInterop
+
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package aries
+
+import (
+	"fmt"
+
+	vdrapi "github.com/hyperledger/aries-framework-go/pkg/framework/aries/api/vdr"
+	"github.com/hyperledger/aries-framework-go/pkg/framework/context"
+	"github.com/hyperledger/aries-framework-go/pkg/vdr"
+	"github.com/hyperledger/aries-framework-go/pkg/vdr/httpbinding"
+	"github.com/hyperledger/aries-framework-go/pkg/vdr/key"
+	"github.com/hyperledger/aries-framework-go/pkg/vdr/peer"
+	"github.com/hyperledger/aries-framework-go/pkg/vdr/sov"
+)
+
+/*
+This file contains interop fixes at the framework level, that break compliance with various RFCs / specs.
+As interop partners continue their interop work, these fixes should stop being necessary.
+*/
+
+func createVDR(frameworkOpts *Aries) error {
+	ctx, err := context.New(
+		context.WithKMS(frameworkOpts.kms),
+		context.WithCrypto(frameworkOpts.crypto),
+		context.WithStorageProvider(frameworkOpts.storeProvider),
+		context.WithServiceEndpoint(serviceEndpoint(frameworkOpts)),
+	)
+	if err != nil {
+		return fmt.Errorf("create context failed: %w", err)
+	}
+
+	var opts []vdr.Option
+
+	p, err := peer.New(ctx.StorageProvider())
+	if err != nil {
+		return fmt.Errorf("create new vdr peer failed: %w", err)
+	}
+
+	// Interop: use a did:sov VDR that wraps the did:peer VDR and httpbinding VDRs,
```

Comment:
Please check that this hasn't already been fixed. I'd rather not have this.

Reply:
did some refactoring and got rid of it :+1: